doc/man3/X509_PUBKEY_new.pod

```diff
@@ -46,6 +46,9 @@ structure defined in RFC5280 and used in certificates and certificate requests.
 
 X509_PUBKEY_new() allocates and initializes an B<X509_PUBKEY> structure.
 
+X509_PUBKEY_dup() creates a duplicate copy of the B<X509_PUBKEY> object
+specified by I<a>.
+
 X509_PUBKEY_free() frees up B<X509_PUBKEY> structure B<a>. If B<a> is NULL
```

Comment:
Please consider changing those incorrect `B<>` to `I<>` for consistency as you're touching the file.

Reply:
Are structure names I<>?  They look like they are B<>.

Reply:
Structure names are `B<>`. I am talking about the `B<a>` and other parameter names below.

Reply:
Ah, okay.

Reply:
Done now.